unity-client/Assets/ABConverter/VisualTests.cs

```diff
@@ -0,0 +1,220 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using DCL.Helpers;
+using System;
+using UnityEditor;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+using UnityEngine.Networking;
+using Object = UnityEngine.Object;
+using Random = UnityEngine.Random;
+
+namespace DCL.ABConverter
+{
+    public static class VisualTests
+    {
+        static readonly string abPath = Application.dataPath + "/../AssetBundles/";
+        static readonly string baselinePath = VisualTestHelpers.baselineImagesPath;
+        static readonly string testImagesPath = VisualTestHelpers.testImagesPath;
+
+        public static IEnumerator TestConvertedAssets(Core core = null, Environment env = null, Action<Core.ErrorCodes> OnFinish = null)
+        {
+            EditorSceneManager.OpenScene($"Assets/ABConverter/VisualTestScene.unity", OpenSceneMode.Single);
+
+            VisualTestHelpers.baselineImagesPath += "ABConverter/";
+            VisualTestHelpers.testImagesPath += "ABConverter/";
+
+            var gltfs = LoadAndInstantiateAllGltfAssets();
+
+            VisualTestHelpers.generateBaseline = true;
+
+            foreach (GameObject go in gltfs)
+            {
+                go.SetActive(false);
+            }
+
+            foreach (GameObject go in gltfs)
+            {
+                go.SetActive(true);
+                yield return VisualTestHelpers.TakeSnapshot($"ABConverter_{go.name}.png", Camera.main, new Vector3(7, 7, 7), Vector3.zero);
```

Comment:
Always positioning the camera at `7,7,7` wouldn't cause issues with big models? Maybe we'd need to position the camera in a dynamic way?